web/plugin-manager/src/pages/Manager.tsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState } from "react";
+import PluginBlock from "../components/pluginBlock";
+import { Plugin, PluginProps } from "../../../shared/interfaces/IPlugin";
+import massaLogoLight from "../assets/MASSA_LIGHT_Detailed.png";
+import axiosServices from "../services/axios";
+import { AxiosResponse } from "axios";
+import alertHelper from "../helpers/alertHelpers";
+import { PuffLoader } from "react-spinners";
+function Manager() {
+    let pluginsInfos: AxiosResponse<any, any> = {} as AxiosResponse<any, any>;
+
+    //State to store error
+    const [error, setError] = useState(<></>);
+    //State to store plugins populated
+    const [pluginsPopulated, setpluginsPopulated] = useState([<PuffLoader/>])
+    //Callback to remove Error
+    function removeError(): void {
+        setError(<></>);
+    }
+    function setErrorHandler(errorType: string, errorMessage: string): void {
+        setError(alertHelper(errorType, errorMessage, removeError));
+        setInterval(() => {
+            setError(<></>);
+        }, 10000);
+    }
+    // Initialize UI on render
+    const initializeUi = async () => {
+        pluginsInfos = await axiosServices.getPluginsInfo();
+        populatePlugins();
+    };
+
+    async function getPluginsInfo () {
+        try {
+            pluginsInfos = await axiosServices.getPluginsInfo();
+            console.log(pluginsInfos)
+            populatePlugins();
+        } catch (error) {
+            setErrorHandler("error", "Get plugins infos failed ");
+        }
+    };
+
+
+    // Update plugin status each 10 seconds
+    // Create a loop to fetch getPluginsInfo and update the status
+    useEffect(() => {
+        initializeUi();
+        const interval = setInterval(async () => {
+            getPluginsInfo();
+        }, 10000);
+        return () => clearInterval(interval);
+    }, []);
+
+    const mock: Plugin = {
+        name: "Plugin 1",
+        logo:
+            "https://upload.wikimedia.org/wikipedia/fr/thumb/1/15/Audi_logo.svg/1280px-Audi_logo.svg.png",
+        description: "If you see this you probably have a problem with the plugin manager",
+        version: "1.0.0",
+        status: "Down",
+        home: "/urlOfPlugin",
+        // isUpdate: true,
+        id: "1",
+    };
+    // Mocks in case we don't have the plugin manager
+    let mocks = [mock];
+
+    function populatePlugins () {
+        console.log(pluginsInfos.status)
+        setpluginsPopulated((pluginsInfos.status == 200)
```

Comment:
difficult to read.

Reply:
For me its the same. 
![image](https://user-images.githubusercontent.com/10938862/218105411-280a5b5c-075f-4f85-b3b1-d04fb63e8b98.png)

Will push the if else. But ternary are not demons.